commit 899b872d519a5e743aee021d249915f0659fa55b
Author:     Isaac Yang <isaacy@nvidia.com>
AuthorDate: Fri May 22 09:52:24 2026 -0700
Commit:     GitHub <noreply@github.com>
CommitDate: Fri May 22 16:52:24 2026 +0000

    [2.8]: Lower a log level on one specific message (#4674)
    
    ### Description
    
    This severity of this log message should be lower.
    
    ### Types of changes
    <!--- Put an `x` in all the boxes that apply, and remove the not
    applicable items -->
    - [x] Non-breaking change (fix or new feature that would not break
    existing functionality).
    - [ ] Breaking change (fix or new feature that would cause existing
    functionality to change).
    - [ ] New tests added to cover the changes.
    - [ ] Quick tests passed locally by running `./runtest.sh`.
    - [ ] In-line docstrings updated.
    - [ ] Documentation updated.
---
 nvflare/private/fed/server/server_engine.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nvflare/private/fed/server/server_engine.py b/nvflare/private/fed/server/server_engine.py
index a6adcbd6..e0003f3d 100644
--- a/nvflare/private/fed/server/server_engine.py
+++ b/nvflare/private/fed/server/server_engine.py
@@ -846,7 +846,7 @@ class ServerEngine(ServerEngineInternalSpec, StreamableEngine):
         data = return_data.payload
         clients = data.get(ServerCommandKey.CLIENTS, None)
         if clients is None:
-            self.logger.error(f"parent failed to return clients info for job {job_id}")
+            self.logger.debug(f"parent failed to return clients info for job {job_id}")
         return clients
 
     def update_job_run_status(self):
